packages/azure-container-services/.eslintignore

```diff
@@ -0,0 +1 @@
+**/node_modules/*
```

Comment:
*Question*: Why do we need this? 🤔 I can see that we ignore the node_modules in other packages too.

https://eslint.org/docs/latest/use/configure/ignore#the-eslintignore-file
Eslint auto ignores node_modules. IMO we dont need this

> node_modules/ is ignored.

But maybe when using mono repos, we need to make use of `ignorePatterns`.

Reply:
oh yes, you are right!. Will update.